src/content/documentation/develop/differences-between-desktop-and-android-extensions.md

```diff
@@ -168,15 +168,17 @@ Permissions to use certain WebExtension APIs must be requested in the `manifest.
 
 {% capture content %}
 
-## Other notes
+## Storage
 
-::: note
-At the time of writing there was an issue with [`storage.sync()`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/storage/sync) and data is not synchronized with the user’s Mozilla account from Firefox for Android. More details can be found in [bug 1316442](https://bugzilla.mozilla.org/show_bug.cgi?id=1316442).
-:::
+Desktop Firefox supports the [local](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/storage/local), [managed](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/storage/managed), [session](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/storage/session), and [sync](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/storage/sync) storage areas.
+
+Firefox for Android supports local, session, and sync storage. However, data in sync storage is not synchronized with the user's Mozilla account from Firefox for Android. More details can be found in [bug 1316442](https://bugzilla.mozilla.org/show_bug.cgi?id=1316442).
```

Comment:
@rpl This bug is closed with a comment, "Closing in favor of the GV Bug." but no references is provided and I can't find anything obvious representing a bug for this. Is it considered a bug?

Reply:
The new tracking bug is [Bug 1625257 - Support |storage.sync| API](https://bugzilla.mozilla.org/show_bug.cgi?id=1625257) (the one that Bug 1316442 has been closed as a duplicate of).